### ISSUE TITLE
"/addons folder not visible

### ISSUE THREAD

**Issue by appsheld** [home-assistant/addons#2608]
### Describe the issue you are experiencing

The Folder addons is not visible in the sidepanel

lrwxrwxrwx    1 root     root             7 Aug  2 09:36 addons -> /addons 

The List starts with folder "Backup".

### What type of installation are you running?

Home Assistant OS

### Which operating system are you running on?

Home Assistant Operating System

### Which add-on are you reporting an issue with?

File editor

### What is the version of the add-on?

5.3.3

### Steps to reproduce the issue

open existing folders in sidepanel
go to "/"

### Anything in the Supervisor logs that might be useful for us?

```txt
no
```

### Anything in the add-on logs that might be useful for us?

```txt
no
```

**Comment by mdegat01** [comment]
Right because it has no access to it:
https://github.com/home-assistant/addons/blob/91e8dc582e75b7759e82441beea8c6165e98c8ee/configurator/config.yaml#L18-L23

That folder is primarily for testing addons during development. Why do you want to see it?

**Comment by appsheld** [comment]
I wanted test my self made local addon and edit the 3 files direct in HASOS, not over ssh and nano.

Thanks

**Comment by mdegat01** [comment]
I assume the reason it isn't exposed for the configurator addon is because that's the new user friendly config editing one. Having addons is confusing for users who aren't HA experts since they expect it to contain data from their addons and it has nothing to do with that.

I would suggest installing the studio code server addon instead if you are doing addon development. It provides a full fledged IDE in browser with syntax highlighting, code completion etc for whatever language you're developing the addon in

**Comment by appsheld** [comment]
The reasoning doesn't convince me. If this is correct, then a corresponding note should be given in the documentation.

**Comment by apbarratt** [comment]
I have to agree, the file editor should explain predominantly in its description that it is severely crippled in access to files it can edit.  I agree that the VS Studio addon is a great asset for this purpose but given it recommends 4GB of ram and (in my experience) causes the home assistant to crash frequently when in use on a Raspbery Pi 3 (for the aforementioned reason I've no doubt), then it is not a viable solution to many users.

Either File Editor should explain that it only provided access to certain folders or a feature for advanced use should be provided that gives total access to the file system from the root level, with the obvious warning that tinkering is done at the users own risk, which I'd have frankly already been assuming to be honest.
